page/partials/header.ejs

```diff
@@ -7,18 +7,18 @@
         <span class="icon-bar"></span>
```

Comment:
Active page should have black color, 
currently it appears as if commit is always active